README.md

```diff
@@ -1,59 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# React Capstone
 
-> One paragraph statement about the project.
+> On this project I built a single page app, that retrieves info from a cryptocurrency API, displays the info on the homepage and allows you to navigate to the details page to see more specific information.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- JavaScript
+- React
+- Redux
+- CSS
+- Jest
+- Reacting Test Library
 
-## Live Demo
+## Live Page
 
-[Live Demo Link](https://livedemo.com)
+[Live Page](https://leolpaz-react-capstone.netlify.app/)
 
+## Demo Video
 
-## Getting Started
+[Video](https://drive.google.com/file/d/1qk3917-XSa0npW9rZGZ7LJPiSnnEVueL/view)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+## Getting Started
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+- Clone the repository with git clone https://github.com/leolpaz/React-Capstone.git;
+- Install dependencies with npm i/yarn install;
+- Use npm start/yarn start to create the dev server;
+- Go to http://localhost:3000/ on your browser if it doesn't do it automatically once you create the dev server;
+- Run "npm/yarn run test" to see if the tests are passing, everything should pass;
 
 
 
 ## Authors
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely.